hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -226,34 +232,58 @@
       desc: "Global escalation causes memory to no longer be accessible."
     }
     { name: "MEM.CTRL.LOCAL_ESC",
-      desc: "A subset of fatal errors cause memory to no longer be accessible."
+      desc: '''
+        A subset of fatal errors cause memory to no longer be accessible.
+        This subset is defined in !!STD_FAULT_STATUS.
```

Comment:
This may not get rendered correctly at the moment... can you quickly build this locally and file an issue if it doesn't?

Reply:
wait really? do you mean it's the `!!` that's the issue? or the the quotation marks?

Reply:
this actually works, pretty neat.